ons_alpha/bulletins/models.py

```diff
@@ -164,3 +175,16 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+    @path("previous/v<int:version>/")
+    def previous_version(self, request, version):
+        try:
+            page_revision = Revision.objects.get(pk=version)
```

Comment:
Issue: There will need to be some additional checks here.  We need to make sure the revision is of the same page type, and that it's published so it's allowed to be viewed publicly.

Also, I _think_ that the version number is meant to be incrementing (ie v1, v2 etc), whereas the versions here will be the ids of the `Revision`, which will increment across all revisions on the site.

Reply:
I believe this has been resolved.

Reply:
It looks like the code still uses the `version` as the `pk` for the `Revision`, so I don't think this is resolved yet.